src/sinol_make/commands/run/__init__.py

```diff
@@ -368,48 +377,98 @@ def print_view(output_file=None):
 		return program_groups_scores
 
 
-	def validate_subtasks(self):
-		print("Validating subtasks...")
-		if 'subtasks' not in self.config.keys():
-			print(util.bold(util.color_red('Subtasks description not defined in config.yml.')))
-			exit(1)
+	def validate_expected_scores(self):
+		print("Validating expected scores...")
+		if 'sinol_expected_scores' not in self.config.keys():
+			print(util.bold(util.color_yellow('Expected scores description not defined in config.yml. ' \
```

Comment:
The "print bold color and exit" pattern shows *very often* in this code, but it's merely an implementation detail and is not worth my attention during review. I suggest to move it to a separate function (e.g. one that produces an error and one that produces a warning). That will shorten the code an allow me to more quickly read the code.

Reply:
I added `util.warning` and `util.error` which return colored strings.